package.json

```diff
@@ -50,6 +52,10 @@
     "angular-aria": "~1.5.x",
     "angular-cookies": "~1.5.x",
     "angular-mocks": "~1.5.x",
+    "angular-translate": "^2.13.1",
```

Comment:
Change these to use tilde ranges like the other dependencies we have. Our tilde ranges only accept patch updates while the NPM default ranges accept minor version updates as well. That's not necessarily a bad thing, but I personally opt (and opted here) for tilde ranges for safety (as the ecosystem can be brittle).